lib/ansible/modules/cloud/docker/docker_image.py

```diff
@@ -533,6 +539,7 @@ def build_image(self):
             tag=self.name,
             rm=self.rm,
             nocache=self.nocache,
+            network_mode=self.network,
```

Comment:
You should only put `network_mode` into `params` if it is not `None`. Otherwise this will fail for a too old docker-py version (i.e. before 2.4.0)